apis/apis-overview/budget-api.md

```diff
@@ -23,37 +23,60 @@ The interval that the budget will reset with (either `weekly` or `monthly`).
 The day the budget will reset. When `interval=weekly`, `intervalDay` is the day of the week, with `intervalDay=0` for Sunday, `intervalDay=1` for Monday, etc. When `interval=monthly`, `intervalDay` corresponds with the day of the month.
 {% endswagger-parameter %}
 
-{% swagger-parameter in="body" name="amount" type="string" %}
+{% swagger-parameter in="body" name="spendLimit" type="int" required="true" %}
 The budget limit value. Currency can be configured in _Settings >_ Cloud Cost Settings > Currency dropdown.
 {% endswagger-parameter %}
 
 {% swagger-parameter in="body" name="id" type="string" %}
-Only should be used when updating a budget rule, ID of the budget rule being modified
+Only should be used when updating a budget rule, ID of the budget rule being modified. For more info, see the [Using the `id` parameter](budget-api.md#using-the-ip-parameter) section below.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="body" name="action" type="[]*BudgetAction" required="false" %} Optional configurations for providing visibility when your budget exceeds a specified percentage threshold. This parameter can generate emails, and Slack or Microsoft Teams messages to suit your work environment. For more information, see the [Using Budget Actions](budget-api.md#Using Budget Actions) section below.
 {% endswagger-parameter %}
 
 {% swagger-response status="200: OK" description="" %}
-```javascript
+```
 {
     "code": 200,
     "data": [
         {
-            "name": "",
-            "id": "",
-            "property": "",
-            "values": [
-                ""
+            "name": "<budget name>",
+            "id": "<budget id>",
+            "values": {
+                "<namespace or cluster>": [
+                    "<name of namespace of cluster>"
+                ]
+            },
+            "kind": "",
+            "interval": "",
+            "intervalDay": ,
+            "spendLimit": ,
+            "actions": [
+                {
+                    "amount": 0,
+                    "percentage": 1,
+                    "slackWebhooks": [],
+                    "msTeamsWebhooks": [],
+                    "emails": [],
+                    "lastFired": ""
+                }
             ],
-            "kind": "soft",
-            "interval": "weekly",
-            "intervalDay": 0,
-            "amount": 0
+            "resources": {
+                "namespaces": [],
+                "clusters": []
+            },
+            "window": {
+                "start": "",
+                "end": ""
+            },
+            "currentSpend":
         }
 }
 ```
 {% endswagger-response %}
 {% endswagger %}
 
-{% swagger method="get" path="/model/budget/recurring/list" baseUrl="http://<your-kubecost-address>" summary="Get recurring budget rule(s)" %}
```

Comment:
there's no concept of recurring anymore, you could drop it to just "budget" -> so it can be Get all  Budgets

Reply:
Good catch, forgot to update this endpoint